paddle/fluid/inference/api/CMakeLists.txt

```diff
@@ -18,8 +18,7 @@ if(APPLE)
 endif(APPLE)
 
 set(inference_deps ${analysis_deps}
-  paddle_inference_api paddle_fluid_api
-  analysis pass naive_executor
+  paddle_inference_api analysis naive_executor
```

Comment:
(suggestion) dont' use string that has not been defined yet

Reply:
Done